include/z64bgcheck.h

```diff
@@ -27,10 +24,30 @@ struct DynaPolyActor;
 #define FUNC_80041EA4_VOID_OUT 12
 
 typedef struct {
-    Vec3f scale;
-    Vec3s rot;
-    Vec3f pos;
-} ScaleRotPos;
+    /* 0x00 */ Vec3f scale;
+    /* 0x0C */ Vec3s rot;
+    /* 0x14 */ Vec3f pos;
+} ScaleRotPos; // size = 0x20
+
+// Macros for `CollisionPoly`
+
+#define COLPOLY_NORMAL_FRAC (1.0f / SHT_MAX)
+#define COLPOLY_SNORMAL(x) ((s16)((x) * SHT_MAX))
+#define COLPOLY_GET_NORMAL(n) ((n)*COLPOLY_NORMAL_FRAC)
+#define COLPOLY_VTX_CHECK_FLAGS_ANY(vI, flags) ((vI) & (((flags)&7) << 13))
+#define COLPOLY_VTX_FLAGS(vI) ((vI)&0xE000)
```

Comment:
```suggestion
#define COLPOLY_VTX_FLAGS_MASK(vI) ((vI)&0xE000)
```
Seems similar to the situation I had with `RELOC_TYPE_MASK` which leaves it unshifted, but just masked. Perhaps it should have an `ED` and be `COLPOLY_VTX_FLAGS_MASKED` 🤔?

Reply:
Note this macro is only used twice, for the very specific purpose of constructing the "real" `flags_vIA`/B values from the original ones stored in objects in the case of dynapoly collision

So sure I can name it however you want, it isn't a huge deal at all. Can also do `#define COLPOLY_VTX_FLAGS_MASK 0xE000` :shrug: 

Also #1359 currently calls the shift-less macros `_NOSHIFT`, but as it hasn't been discussed at all idk if it's worth to reference it

Reply:
unrelated to the topic of conversation, but space out the &'s pls (`((vI)&0xE000)` -> `((vI) & 0xE000)`)

Reply:
Tbh I kind of like `MASKED` along with changing `RELOC_TYPE_MASK`, but as long as the two are consistent it should be fine. (comment `RELOC_TYPE_MASK` macro https://github.com/zeldaret/oot/pull/1235#discussion_r884032332 for reference.)

Reply:
After much internal conflicting, changed to `COLPOLY_VTX_FLAGS_MASKED`